<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maska</class>
 <widget class="QMainWindow" name="Maska">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>402</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Maska</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3" rowspan="5">
     <widget class="QGraphicsView" name="mapViewConst">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="decSpeed">
      <property name="text">
       <string>Speed -</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="addSpeed">
      <property name="text">
       <string>Speed +</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="nextTurn">
      <property name="text">
       <string>Next Turn</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="reset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="autoNextTurn">
      <property name="text">
       <string>Auto Next Turn</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="addCar">
      <property name="text">
       <string>Car +</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="deleteCar">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Car -</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRandom">
    <property name="title">
     <string>Random</string>
    </property>
    <addaction name="actionRandomALL"/>
    <addaction name="actionRandom_unset_destinations"/>
    <addaction name="actionSticky_Random"/>
   </widget>
   <addaction name="menuRandom"/>
  </widget>
  <action name="actionRandomALL">
   <property name="text">
    <string>Random all destinations</string>
   </property>
  </action>
  <action name="actionRandom_unset_destinations">
   <property name="text">
    <string>Random unset destinations</string>
   </property>
  </action>
  <action name="actionSticky_Random">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Random</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
